modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/process/kaleo_draft_definition_action.jsp

```diff
@@ -51,7 +53,9 @@ KaleoDefinitionVersion kaleoDefinitionVersion = (KaleoDefinitionVersion)row.getO
 		<portlet:param name="name" value="<%= kaleoDefinitionVersion.getName() %>" />
 	</portlet:actionURL>
 
-	<liferay-ui:icon-delete
-		url="<%= deleteURL %>"
-	/>
+	<c:if test="<%= permissionChecker.isCompanyAdmin() %>">
```

Comment:
Hey @MariMalta, please move the `portlet:actionURL` declaration to the if statement as well.

Thx